packages/ui/src/primitives/tailwind/Table/index.tsx

```diff
@@ -57,7 +57,7 @@ const TableHeadRow = ({
   children: JSX.Element | JSX.Element[]
 }) => {
   const twClassName = twMerge('text-left capitalize', className)
-  const twClassNameThead = twMerge('sticky top-0 z-10 bg-theme-table-secondary', theadClassName)
+  const twClassNameThead = twMerge('sticky top-[-2px] z-10 bg-theme-table-secondary', theadClassName)
```

Comment:
why did we add -2px?
(i think it should be a page layout that would be causing the issue if it has moved downward)

Reply:
this is because now the header is fixed with the vertical scroll and the rows appears a bit above the the header

Reply:
@CITIZENDOT your thoughts?